.pre-commit-config.yaml

```diff
@@ -35,15 +35,15 @@ repos:
     hooks:
       - id: codespell
   - repo: https://github.com/crate-ci/typos
-    rev: v1.28.4
+    rev: dictgen-v0.3.1
```

Comment:
This looks weird but I think it's fine.

Reply:
yeah pre-commit doesn't care what you label your tags in https://github.com/crate-ci/typos/tags
feels dangerous, but don't think we need to care